src/interfaces/IRedemption.sol

```diff
@@ -26,6 +26,12 @@ interface IRedemption {
         address indexed redeemer, address indexed to, uint256 superstateTokenInAmount, uint256 usdcOutAmount
     );
 
+    /// @dev Event emitted when SUPERSTATE_TOKEN is redeemed for USDC
+    /// @param redeemer The address of the entity redeeming
+    /// @param superstateTokenInAmount The amount of SUPERSTATE_TOKEN to redeem
+    /// @param usdcOutAmount The amount of USDC the redeemer gets back
+    event Redeem(address indexed redeemer, uint256 superstateTokenInAmount, uint256 usdcOutAmount);
```

Comment:
shouldnt this be redeemv2?

Reply:
I decided to keep the [old event along with the new one](https://github.com/superstateinc/onchain-redemptions/blob/khail/avoid-redeem-breaking-changes/src/interfaces/IRedemption.sol#L20-L33) it made the contract inheritance easier in the testing since the updated interface would be backwards compatible.

Reply:
I don't think this is the right approach. When we upgrade the contract the original Redeem event wont be emitted from it anymore, so leaving it in the interface is misleading

Reply:
https://github.com/superstateinc/onchain-redemptions/pull/37/commits/fef496a3fff35627318501fb6536be627a64a65c